<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProfileDialog</class>
 <widget class="QDialog" name="NewProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Name</string>
       </property>
       <property name="buddy">
        <cstring>lineEditName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>This is a friendly name to distinguish the connection</string>
         </property>
         <property name="placeholderText">
          <string>User friendly unique connection name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxCustomize">
         <property name="text">
          <string>&amp;Customize</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelGateway">
       <property name="text">
        <string>Gateway</string>
       </property>
       <property name="buddy">
        <cstring>lineEditGateway</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditGateway">
       <property name="toolTip">
        <string>Specify the hostname to connect to; a port may be specified after the host separated with a colon ':'</string>
       </property>
       <property name="placeholderText">
        <string>https://my_server[:443]/[usergroup]</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelNote">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Note: you can anytime later edit profile details...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save|QDialogButtonBox::SaveAll</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditGateway</tabstop>
  <tabstop>checkBoxCustomize</tabstop>
  <tabstop>lineEditName</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProfileDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProfileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxCustomize</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>textChanged(QString)</slot>
 </slots>
</ui>
